src/CodeStyle/Core/Analyzers/Formatting/FormatterHelper.cs

```diff
@@ -77,6 +78,23 @@ internal static async Task<SyntaxTree> FormatAsync(SyntaxTree syntaxTree, ISynta
         public static SyntaxNode Format(SyntaxNode node, ISyntaxFormattingService syntaxFormattingService, OptionSet options, CancellationToken cancellationToken)
             => Format(node, syntaxFormattingService, SpecializedCollections.SingletonEnumerable(node.FullSpan), options, rules: null, cancellationToken: cancellationToken);
 
+        /// <summary>
+        /// Formats the whitespace of a syntax tree.
+        /// </summary>
+        /// <param name="node">The root node of a syntax tree.</param>
+        /// <param name="annotation">The descendant nodes of the root to format.</param>
+        /// <param name="options">An optional set of formatting options. If these options are not supplied the current set of options from the workspace will be used.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>The formatted tree's root node.</returns>
+        public static SyntaxNode Format(SyntaxNode node, ISyntaxFormattingService syntaxFormattingService, SyntaxAnnotation annotation, OptionSet options, IEnumerable<AbstractFormattingRule> rules, CancellationToken cancellationToken)
+        {
+            var spans = (annotation == SyntaxAnnotation.ElasticAnnotation)
```

Comment:
```suggestion
            var spans = annotation == SyntaxAnnotation.ElasticAnnotation
```

Reply:
This is existing code that I just moved. I am hoping to instead enforce this rule in CI for Workspaces/Features layer once moved to CodeStyle NuGet package.